ground/src/main/java/com/google/android/ground/repository/SurveyRepository.kt

```diff
@@ -78,6 +79,9 @@ constructor(
     return survey
   }
 
+  private fun maybeGetOfflineSurvey(surveyId: String): @Cold Maybe<Survey> =
```

Comment:
I don't think we need to differentiate null/empty in the method names `maybeGetOfflineSurvey`. Once migrated to coroutines this will just be a suspend fun that returns `Survey?`. Or is there precedent for this?